app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/controls/filtersControl/filtersControl.jsx

```diff
@@ -14,13 +14,14 @@ import {
   loadingSelector,
   filtersPaginationSelector,
 } from 'controllers/filter';
-import { WIDGET_WIZARD_FORM } from '../../../widgetWizardContent/wizardControlsSection/constants';
+import { PAGE_KEY, SIZE_KEY } from 'controllers/pagination';
+import { WIDGET_WIZARD_FORM } from '../../../../widgetWizardModal/constants';
```

Comment:
I think absolute path will be better here

Reply:
Fixed